properties/basic.py

```diff
@@ -797,8 +799,10 @@ def validate(self, instance, value):
     @property
     def info(self):
         info = 'a unicode string' if self.unicode else 'a string'
-        if self.regex is not None and hasattr(self.regex, 'pattern'):
-            info += ' that matches pattern "{}"'.format(self.regex.pattern)    #pylint: disable=no-member
+        if self.regex is not None:
+            info += ' that contains pattern'
```

Comment:
Same as above; still "matches", I think? Don't know what your feelings on it are, @fwkoch?